core/spec/models/spree/order/checkout_spec.rb

```diff
@@ -184,6 +185,28 @@ def assert_state_changed(order, from, to)
         expect(order.total).to eq(10.5)
       end
 
+      it 'updates prices' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.